src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -25,12 +25,8 @@
  */
 package org.ow2.proactive.connector.iaas.cloud.provider.azure;
 
+import java.util.*;
```

Comment:
That's a very greedy importation !

Reply:
I agree, but it was automatically generated :/

Reply:
Fixed!